atm.js

```diff
@@ -1,3 +1,75 @@
 $(document).ready(function(){
 // Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+var savings = {
+  balance: $("#savings .balance"),
+  putted: $("#savings .input"),
+  draw: $("#savings .withdraw").click( function(){
+    if ( (savings.balVal - parseInt(savings.putted.val()) ) >= 0){
+      savings.balVal = savings.balVal - parseInt(savings.putted.val());
+      savings.balance.html("$" + savings.balVal.toString());
+      savings.putted.val("");
+
+      if(savings.balVal == 0){
+        checking.balance.addClass("zero");
+      }else{
+        checking.balance.removeClass("zero")
+      }
```

Comment:
You could shorten this by using `.toggleClass`. http://api.jquery.com/toggleclass/